README.md

```diff
@@ -156,6 +158,22 @@ require('nordic').setup({
 </details>
 
 
+<details>
+    <summary><b><code>after_palette</code></b></summary>
+&nbsp;
+
+An example of setting the visual selection color (for more values see [this file](https://github.com/AlexvZyl/nordic.nvim/blob/main/lua/nordic/colors/init.lua)):
+```lua
+require('nordic').setup({
+    after_palette = function(palette)
+        palette.bg_visual = require("nordic.utils").blend(palette.orange.base, palette.bg, 0.15)
```

Comment:
Can we just change this to:
```lua
local U = require("nordic.utils")
palette.bg_visual = U.blend(palette.orange.base, palette.bg, 0.15)
```

No other comment.

Reply:
Sure